dashboard/lib/tasks/asset_sync.rake

```diff
@@ -54,7 +54,7 @@ Rerun `assets:precompile` to regenerate new assets and try again."
   # updated if there are any changes to Sprockets processors.
   module NoDoubleDigest
     def digest_path
-      logical_path.match?(/wp\h{32}/) ? logical_path : super
+      logical_path.match?(/wp\h{20}/) ? logical_path : super
```

Comment:
@wjordan , I am particularly interested in your opinion on this aspect of the plan, where we tell sprockets to skip digesting for assets which already have webpack content hashes.